frb_rust/src/thread.rs

```diff
@@ -1,4 +1,32 @@
-pub const WORKERS_COUNT: usize = 4;
+#[cfg(not(wasm))]
+fn get_thread_count() -> usize {
+    #[allow(unused_mut, unused_assignments)]
+    let mut thread_count: usize = 4;
+    #[cfg(feature = "thread-count-single")]
+    {
+        worker_count = 1;
+    }
+    #[cfg(feature = "thread-count-max")]
+    {
+        thread_count = std::thread::available_parallelism().unwrap().get();
+    }
+    thread_count
+}
+
+#[cfg(wasm)]
+fn get_worker_count() -> usize {
+    #[allow(unused_mut, unused_assignments)]
+    let mut worker_count: usize = 4;
+    #[cfg(feature = "worker-count-single")]
+    {
+        worker_count = 1;
+    }
+    #[cfg(feature = "worker-count-max")]
+    {
+        worker_count = 16;
+    }
+    worker_count
+}
```

Comment:
uber-nit: looks like we can merge the two into one function, and only use `cfg(wasm)` for a few lines, since they only have minor differences

Reply:
Good point, I'll be working on that in a day

Reply:
Take your time!